PWGHF/TableProducer/candidateCreator3Prong.cxx

```diff
@@ -635,6 +641,14 @@ struct HfCandidateCreator3ProngExpressions {
         continue;
       }
 
+      if (useTimeFrameBorderCut) { // accept only mc particles coming from bc that are far away from TF border and ITSROFrame
+        auto bc = mcCollision.bc_as<BCsInfo>();
+        if (!bc.selection_bit(o2::aod::evsel::kNoITSROFrameBorder) || !bc.selection_bit(o2::aod::evsel::kNoTimeFrameBorder)) {
+          rowMcMatchGen(flag, origin, channel);
+          continue;
+        }
+      }
+
```

Comment:
In MC simulation, the lower track-reconstruction efficiency near the ITS ROF border is not reproduced. So, there is no need to reject collisions in that region, both at generated and reconstructed level. I would keep only the cut on the TF border for the moment. In general, it is also good to separate these selections so that they can be applied independently.